docs/6.x/pack.md

```diff
@@ -912,3 +912,38 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Direct Upgrades From Older LTS Versions
+
+Kubernetes has a strict version skew [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew) which makes
+cluster upgrades to versions of Kubernetes being more than a couple of minor versions apart complicated if not impossible, hence
+upgrading such a cluster to the version of Kubernetes that would break the version skew policy is not recommended. This can become a big issue if
+clusters are not kept up-to-date.
+
+To correctly upgrade such a cluster, it first needs to be upgraded to a cluster image with a compatible version of Kubernetes (according to the version
+skew policy). This step should be repeated until the cluster has been upgraded to the target version.
+
+For example, to upgrade a cluster based on Gravity `5.0.35` (which is based on Kubernetes `1.9.13`) to a cluster based on Gravity `5.5.21` (which
+is based on Kubernetes `1.13.11`), the cluster needs to be upgraded to Gravity version `5.2.15` (with Kubernetes `1.11.9`) first before upgrading
+to `5.5.21`.
+
+Since version `5.5.21` `tele` is capable of producing cluster image tarballs that can upgrade such clusters directly by embedding all the required
```

Comment:
This text seems to be embedded twice, once as a doc item, and also as a note. I think I would just leave it as a note that can be removed at some point when it's no longer relevant.